doc/how-to/install.md

```diff
@@ -1,6 +1,76 @@
 (howto-install)=
 # How to install MicroCloud
 
+(pre-deployment-requirements)=
+## Pre-deployment requirements
+
+````{tabs}
+```{group-tab} General
+
+The requirements in this section apply to all MicroCloud deployments.
+
+A physical or virtual machine intended for use as a MicroCloud cluster member must meet the following prerequisites:
+
+- Software: 
+  - Ubuntu 22.04 or newer (LTS version recommended)
+  - snapd 2.59 or newer
+
+- Networking:
+  - Fixed IP addresses (DHCP not supported)
+  - Two network interfaces per cluster member, one for intra-cluster communication and one for external connectivity to the uplink network
+    - Partially or fully aggregated networking setups require more interfaces; see: {ref}`howto-ceph-networking`
+  - Uplink network must support both broadcast and multicast
+  - Intra-cluster interface must have IPs assigned; external connectivity interface must not have any IPs assigned
+
+- Storage:
+  - Disks should be free of existing partitions or file systems
+  - For local storage, each cluster member must have at least one local disk
+  - If you intend to use full disk encryption on a cluster member, it must have `snapd` version `2.59.1` or newer installed and the `dm-crypt` kernel module available
+    - To check if the module exists, run:
+
+    ```
+    sudo modinfo dm-crypt
+    ```
+```
+
+```{group-tab} Testing or development environments
+
+- Physical or virtual machines can be used
+- Minimum cluster size:
+  - 1 member
+- Memory:
+  - Minimum 8 GiB RAM per cluster member
+- Storage:   
+  - If high availability is required, use distributed storage with:
+    - a minimum of 3 cluster members
+    - a minimum of 3 separate disks located across 3 different members
+  - Otherwise, local storage is sufficient
+```
+
+```{group-tab} Production environments
+
+- Physical machines only (no VMs)
+- Minimum cluster size:
+  - 3 members
+  - For critical deployments, we recommend a minimum of 4 members
+- Memory:
+  - Minimum 32 GiB RAM per cluster member
```

Comment:
I guess this is mainly coming from Ceph? Should we mention this or even link to https://docs.ceph.com/en/latest/start/hardware-recommendations/#hardware-recommendations.

Reply:
Should the 8GiB statement from https://canonical-microcloud.readthedocs-hosted.com/en/latest/microcloud/reference/ be removed now as it's stated here?

Maybe on the `reference/` page we can link to the Ceph doc as on there I think the idea is to be more specific.

Reply:
> Should the 8GiB statement from https://canonical-microcloud.readthedocs-hosted.com/en/latest/microcloud/reference/ be removed now as it's stated here?

Yes, I will update that page separately. It needs several updates.

> Maybe on the `reference/` page we can link to the Ceph doc as on there I think the idea is to be more specific.
The reference/ page already links to the Ceph docs.